README.md

```diff
@@ -139,6 +139,8 @@ Here are the main configuration keys.
   "pump.io".
 * *owner* Name of owning entity, if you want to link to it.
 * *ownerURL* URL of owning entity, if you want to link to it.
+* *mainImage* A image/icon for main page, could be a external URL or 
```

Comment:
Since #1496 will land soon this entire section will be gone soon - can you send an MR to https://gitlab.com/pump.io/pump-docs with this change? And then get rid of these two lines in this PR?